doc/Protocols/pr-mserv_x-road_protocol_for_management_services.md

```diff
@@ -31,28 +31,27 @@ Doc. ID: PR-MSERV
 | 07.06.2017 | 1.10    | Additional signature algorithms supported                                | Kristo Heero       |
 | 06.03.2018 | 1.11    | Added terms section, term doc reference and link, fixed references       | Tatu Repo          |
 | 06.02.2019 | 1.12    | Update *clientReg* message description                                   | Petteri Kivimäki   |
-
-## Table of Contents
-
-<!-- vim-markdown-toc GFM -->
-
-* [License](#license)
-* [1 Introduction](#1-introduction)
-    * [1.1 Terms and abbreviations](#11-terms-and-abbreviations)
-    * [1.2 References](#12-references)
-* [2 Format of the Messages](#2-format-of-the-messages)
-    * [2.1 clientReg - Security Server Client Registration](#21-clientreg---security-server-client-registration)
-    * [2.2 clientDeletion - Security Server Client Deletion](#22-clientdeletion---security-server-client-deletion)
-    * [2.3 authCertReg - Security Server Authentication Certificate Registration](#23-authcertreg---security-server-authentication-certificate-registration)
-    * [2.4 authCertDeletion - Security Server Authentication Certificate Deletion](#24-authcertdeletion---security-server-authentication-certificate-deletion)
-* [Annex A Example messages](#annex-a-example-messages)
-    * [A.1 clientReg](#a1-clientreg)
-    * [A.2 clientDeletion](#a2-clientdeletion)
-    * [A.3 authCertReg](#a3-authcertreg)
-    * [A.4 authCertDeletion](#a4-authcertdeletion)
-* [Annex B WSDL File for Management Services](#annex-b-wsdl-file-for-management-services)
-
-<!-- vim-markdown-toc -->
+| 03.06.2019 | 1.13    | Add ownerChange management service                                       | Ilkka Seppälä      |
+
+## Table of Contents- [X-Road: Protocol for Management Services](#x-road-protocol-for-management-services)
```

Comment:
The document title is not needed in this header - other protocol documents do not contain it here either.